.envrc.template

```diff
@@ -9,3 +9,6 @@ export API_V3_URL="https://api-v3.mbta.com"
 # export AWS_SECRET_ACCESS_KEY=
 
 export SECRET_KEY_BASE="local_secret_key_base_at_least_64_bytes_________________________________"
+
+# export TRIP_UPDATES_URL=
+# export VEHICLE_POSITIONS_URL=
```

Comment:
Very nitpicky: It might be helpful to add the link to [this section](https://github.com/mbta/gtfs-documentation/blob/master/reference/gtfs-realtime.md#enhanced-json-feeds) here for reference when a dev is onboarding. Cora's comment is actually my first time seeing this.

Reply:
[np!](https://github.com/mbta/screens/compare/251e06a912af136a72606c623d7066023a42c067..0c9ca14d32f20fa5ad47b1999608bc9af8b7404f)